### PR TITLE
add multicluster resource permissions to console role

### PR DESCRIPTION
**Pull request by lake-of-dreams**
# Description

Please include a summary of the change and which issue is fixed.  If there are any dependencies, for example on a PR in another repository, please list those.

Fixes VZ-2269

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano/templates/06-verrazzano-api.yaml [hunk 1]
```diff
@@ -7,6 +7,19 @@ metadata:
     rbac.authorization.kubernetes.io/autoupdate: "true"
   name: {{ .Values.api.clusterRole }}
```
Comment (wmhopkins) [inline platform-operator/helm_config/charts/verrazzano/templates/06-verrazzano-api.yaml:8]: This role should no longer be needed. Instead, users with one of the verrazzano user roles will be able to see the resources they have access to.

**Review by wmhopkins — COMMENTED** [review]
I think the `{{ .Values.api.clusterRole }}` is not needed at all any more, nor is the binding `{{ .Values.api.clusterRole }}-binding`. A user that is assigned a valid verrazzano role will be able to access the necessary resources.

We probably do need to update keycloak.json to remove the console user and console group, and add verrazzano user to the verrazzano-admins group (name of the group is determined at install time).

File: platform-operator/helm_config/charts/verrazzano/templates/06-verrazzano-api.yaml (hunk 1 above)
Comment (lake-of-dreams) [reply platform-operator/helm_config/charts/verrazzano/templates/06-verrazzano-api.yaml]: fixed

File: platform-operator/helm_config/charts/verrazzano/values.yaml [hunk 2]
```diff
@@ -94,9 +94,7 @@ api:
   imageVersion: 0.32-20210122132046-a84f6166a
   pullPolicy: Always
   port: 8775
-  clusterRole: console
-  consoleUserGroup: console-users
-  consoleUser: console-user
+  impersonatorRoleName: imperosnate-api-user
```
Comment (wmhopkins) [inline platform-operator/helm_config/charts/verrazzano/values.yaml:97]: typo, should be "impersonate-api-user"

File: platform-operator/scripts/install/4-install-keycloak.sh [hunk 3]
```diff
@@ -69,8 +69,9 @@ function install_keycloak {
   # Replace strings in keycloak.json file
   VZ_PW_SALT=$(kubectl get secret -n ${VERRAZZANO_NS} verrazzano -o jsonpath="{.data.salt}")
   VZ_PW_HASH=$(kubectl get secret -n ${VERRAZZANO_NS} verrazzano -o jsonpath="{.data.hash}")
+  VZ_ADMIN_GROUP=$(helm show values ${VZ_CHARTS_DIR}/verrazzano | grep -A3 "userrolebindings:" | grep -A3 "admin:" | tail -n1 | awk '{ print $2}')
```
Comment (wmhopkins) [inline platform-operator/scripts/install/4-install-keycloak.sh:72]: FYI, this code will have to change based on changes I'm making to how role bindings are installed, but no reason to hold up this PR for it.

**Review by wmhopkins — APPROVED** [review]
Looks good to me. There is one typo you should fix, and I think the "-binding" suffix on role binding names is unnecessary (the Kind tells you that it's a binding), but other than that it looks good. Approved pending a fix for the typo.

File: platform-operator/helm_config/charts/verrazzano/values.yaml (hunk 2 above)
Comment (lake-of-dreams) [reply platform-operator/helm_config/charts/verrazzano/values.yaml]: fixed

**Review by wmhopkins — APPROVED** [review]
Ship it!

**Pull request merged**